baselines/spacy_tagging.py

```diff
@@ -0,0 +1,116 @@
+# Lint as: python3
+"""Example tagging for Toxic Spans based on Spacy.
+
+Requires:
+  pip install spacy sklearn
+
+Install models:
+  python -m spacy download en_core_web_sm
+
+"""
+
+import ast
+import csv
+import random
+import statistics
+import sys
+
+import sklearn
+import spacy
+
+sys.path.append('../evaluations')
+import semeval2021
+import fix_spans
+
+def spans_to_ents(doc, spans, label):
+  """Converts span indicies into spacy entity labels."""
+  started = False
+  left, right, ents = 0, 0, []
+  for x in doc:
+    if x.pos_ == 'SPACE':
+      continue
+    if spans.intersection(set(range(x.idx, x.idx + len(x.text)))):
+      if not started:
+        left, started = x.idx, True
+      right = x.idx + len(x.text)
+    elif started:
+      ents.append((left, right, label))
+      started = False
+  if started:
+    ents.append((left, right, label))
+  return ents
+
+
+def read_datafile(filename):
+  """Reads csv file with python span list and text."""
+  data = []
+  with open(filename) as csvfile:
+    reader = csv.DictReader(csvfile)
+    count = 0
+    for row in reader:
+      fixed = fix_spans.fix_spans(
```

Comment:
Shouldn't it be fix_spans._fix_spans instead of fix_spans.fix_spans?

Reply:
Renamed to make public, as it should be.